Super_Simple_RGB_WiFi_Lamp/Super_Simple_RGB_WiFi_Lamp.ino

```diff
@@ -149,6 +152,17 @@ uint8_t visualiserFadeUp                    = 32;
 uint8_t visualiserFadeDown                  = 32;
 uint8_t visualiserHueOffset                 = 170;
 
+//colorWipe
+int wipePosition = -1;
+bool TurningOn = true;
+int colorWipeRed   = 255;
+int colorWipeGreen = 0;
+int colorWipeBlue  = 255;
+int wipeSpeed = 20;
```

Comment:
make sure each variable for the individual modes has the mode name at the start of the variable. ie; colourWipeTurningOn

Reply:
Ok, fair enough.